proto/tabletmanagerdata.proto

```diff
@@ -670,6 +670,7 @@ message VDiffCoreOptions {
   int64 max_extra_rows_to_compare = 7;
   bool update_table_stats = 8;
   int64 max_diff_seconds = 9;
+  bool do_not_start = 10;
```

Comment:
That's what I started with, but the problem then is that if you do not specify the field — which older clients would not do — then the default/zero value is false and it's not started.

Reply:
`autoStart := subFlags.Bool("auto_start", true, "If false, streams will start in the Stopped state and will need to be explicitly started")`

Shouldn't this definition, from vrep workflows, work? Default is `true` and user needs to specify `--auto_start=false` to start vdiff in a stopped state.

Reply:
This is an issue at the proto / RPC layer. Any client sending an older request object will not include that field. So it will be false (the zero value) on the server side

Reply:
I'll try making it optional so that it's a *bool. Then we start it unless a value of false was specifically provided (would be nil in older client requests).

Reply:
Made that change here: https://github.com/vitessio/vitess/pull/16654/commits/fc0cab649562951371d5f58c74dd8a1e84a8b33a 

I'm good either way, so let's go with that if you prefer it.

Reply:
Made that change here: https://github.com/vitessio/vitess/pull/16654/commits/0020b0f86563c837cc74ba162069311795b217a7

I'm good either way, so let's go with that if you prefer it.